instructure/dress_code
# Taking out container div around code output

I've forked off of the dress_code gem to give us more flexibility in with our html when the style guide is generated. 

Test Plan:
- pre bundling - on your local canvas go to /styleguide and inspect the code sample, which should look like this:
  ![screen shot 2014-08-18 at 4 19 10 pm](https://cloud.githubusercontent.com/assets/178799/3958994/f8a74bb4-2725-11e4-9745-71f106d0c388.png)
  pay attention to the tag: `<div class="code-demo">`
- now bundle install
- run `bundle exec rake css:styleguide` to recompile the styleguide
- inspect that same area and now that container should be gone and it should look like:
  ![screen shot 2014-08-18 at 4 24 18 pm](https://cloud.githubusercontent.com/assets/178799/3959040/6e6a29fc-2726-11e4-99d9-9bd99e3fcdd0.png)
  wherein there is no container surrounding it
